python/ray/tune/result_grid.py

```diff
@@ -83,27 +83,13 @@ def __init__(
             self._trial_to_result(trial) for trial in self._experiment_analysis.trials
         ]
 
-    @property
-    def _local_path(self) -> str:
-        """Return path pointing to the experiment directory on the local disk."""
-        return self._experiment_analysis._local_path
-
-    @property
-    def _remote_path(self) -> Optional[str]:
-        """Return path pointing to the experiment directory on remote storage."""
-        return self._experiment_analysis._remote_path
-
     @property
     def experiment_path(self) -> str:
         """Path pointing to the experiment directory on persistent storage.
 
         This can point to a remote storage location (e.g. S3) or to a local
-        location (path on the head node).
-
-        For instance, if your remote storage path is ``s3://bucket/location``,
-        this will point to ``s3://bucket/location/experiment_name``.
-        """
-        return self._remote_path or self._local_path
+        location (path on the head node)."""
+        return self._experiment_analysis.experiment_path
```

Comment:
This is a little different now. It's always returning a filesystem-qualified path. NOT a URI. This is in line with https://github.com/ray-project/ray/issues/38568

cc: @ericl

Reply:
Yup thx! Will use this in the followup PR.